src/Domain/Insights/FixerDecorator.php

```diff
@@ -138,17 +136,10 @@ public function addDiff(string $file, string $diff): void
 
     private function skipFilesFromExcludedFiles(\SplFileInfo $file): bool
     {
-        $path = $file->getFileInfo()->getRealPath();
-        if ($path === false) {
-            return false;
-        }
-        foreach ($this->excludedFiles as $excludedFile) {
-            if (Paths::areEqual($this->dir . DIRECTORY_SEPARATOR . $excludedFile, $path)) {
-                return true;
-            }
-        }
-
-        return false;
+        return array_key_exists(
+            (string) $file->getRealPath(),
+            $this->exclude
+        );
```

Comment:
Does this actually work in the case where real path returns false?

And cool, this is much cleaner than before and prob. much faster, nice work

I have a small suggestion 
```
$path = $file->getRealPath();
return $path !== false && isset($this->exclude[$path]);
```
`isset` is faster, however not a difference of any significance